docs/en/observability/logs-ecs-application.asciidoc

```diff
@@ -0,0 +1,206 @@
+        {
+          "names": ["filebeat-*"],
+          "privileges": ["manage"]
+        }
+      ]
+    }
+  }
+}
+----
++
+Refer to {filebeat-ref}/beats-api-keys.html[Grant access using API keys] for more information.
+
+[discrete]
+[[step-3-ecs-configure-filebeat]]
+=== Step 3: Configure {filebeat}
+
+Add the following configuration to your `filebeat.yaml` file to start collecting log data.
+
+[source,yaml]
+----
+filebeat.inputs:
+- type: filestream  <1>
+  enabled: true
+  paths: /path/to/logs.log  <2>
+----
+<1> Reads lines from an active log file.
+<2> Paths that you want {filebeat} to crawl and fetch logs from.
+
+[discrete]
+[[step-4-ecs-set-up-and-start-filebeat]]
+=== Step 4: Set up and start {filebeat}
+
+From the {filebeat} installation directory, set the {ref}/index-templates.html[index template] by running the command that aligns with your system:
+
+include::tab-widgets/logs/filebeat-setup-widget.asciidoc[]
+
+From the {filebeat} installation directory, start filebeat by running the command that aligns with your system:
+
+include::{beats-root}/libbeat/docs/tab-widgets/start-widget-filebeat.asciidoc[]
+
+[discrete]
+[[ingest-ecs-logs-with-agent]]
+=== Ingest logs with {agent}
+
+Add the custom logs integration to ingest and centrally manage your logs using {agent} and {fleet}:
+
+[discrete]
+[[step-1-add-the-custom-logs-integration-to-your-project-ecs]]
+=== Add the custom logs integration to your project
+
+To add the custom logs integration to your project:
+
+. From your project, navigate to **Project Settings →   Integrations**.
+. Type `custom` in the search bar and select **Custom Logs**.
+. Click **Install {agent}** at the bottom of the page, and follow the instructions for your system to install the {agent}.
+. After installing the {agent}, click **Save and continue** to configure the integration from the **Add Custom Logs integration** page.
+. Give your integration a meaningful name and description.
+. Add the **Log file path**. For example, `/var/log/your-logs.log`.
+. Click **Advanced options**.
+. In the **Processors** text box, add the following YAML configuration to add processors that enhance your data. See {filebeat-ref}/filtering-and-enhancing-data.html[processors] to learn more.
++
+[source,yaml]
+----
+processors:
+  - add_host_metadata: \~
+  - add_cloud_metadata: \~
+  - add_docker_metadata: \~
+  - add_kubernetes_metadata: \~
+----
+. Under **Custom configurations**, add the following YAML configuration to collect data.
++
+[source,yaml]
+----
+  json:
+    overwrite_keys: true <1>
+    add_error_key: true <2>
+    expand_keys: true <3>
+    keys_under_root: true <4>
+  fields_under_root: true <5>
+  fields:
+    service.name: your_service_name [^6]
+    service.version: your_service_version [^6]
+    service.environment: your_service_environment [^6]
+----
++
+<1>: Values from the decoded JSON object overwrite the fields that {agent} normally adds (type, source, offset, etc.) in case of conflicts.
```

Comment:
I think you might need line breaks between the callouts here because they are rendering as one long paragraph.

![image](https://github.com/elastic/observability-docs/assets/14206422/da55bbc3-dd83-4a17-91e4-7d9c7a1da2bd)